models.tsx

```diff
@@ -1,18 +1,22 @@
+import { User as FBUser } from 'firebase/auth'
+
+export interface User extends FBUser {
```

Comment:
Hmmm, is this a thing? I feel like the User response from the server doesn't extend FBUser. Am I confused about something?

Reply:
Ouch actually yeah this is bad lol. Not sure what I was thinking exactly when I did this. 

I did this as a way to cheat into not having to say whether we're dealing with a User from our API or a User from Firebase down when dealing with the Owner below. But really we should have our own `BrackendUser` interface or something. Lemme fix this up

Reply:
@probicheaux I made a `BracketUser` type to represent users from our API, and then we'll use the Firebase User type when looking at the current user context: https://github.com/probicheaux/ReactBracket/pull/18/commits/b1189b7eaff42739212daab20a8b294e549a9e04